pepper-apis/src/main/java/org/broadinstitute/ddp/util/StudyDataLoader.java

```diff
@@ -419,16 +553,25 @@ public ActivityInstanceDto createActivityInstance(JsonElement surveyData, String
 
         long activityInstanceId = dto.getId();
 
+        if (activityCode.toLowerCase().contains("prion") && ddpLastUpdatedAt.equals(ddpCreatedAt)) {
+            //In Gen2 Prion, sometimes the last updated time doesn't actually get updated
+            ddpLastUpdatedAt++;
```

Comment:
It's probably safer to do this date++ in export file generation rather than during data loading.